app/config/config.yml

```diff
@@ -114,12 +114,12 @@ swiftmailer:
     spool:     { type: memory }
 
 # Guzzle configuration
-csa_guzzle:
-    profiler:
-        enabled: "%kernel.debug%"
+love_oss_guzzle5:
```

Comment:
Seen with @eternoendless and @PierreRambaud : this is a BC break as it will fail for people using this configuration key.

Example in this PR: https://github.com/PrestaShop/PrestaShop/pull/13761/files#diff-95a1b476935b61587950fe999d326bbaR165
If someone was using `$clientConfig = $config['csa_guzzle']['clients']['addons_api']['config'];` it will fail after this PR